SensusUI/Inputs/LabelSliderInput.cs

```diff
@@ -0,0 +1,266 @@
+                            _incrementalValue = newIncrementalValue;
+                            _incrementalValueHasChanged = true;
+                            Complete = Value != null;
+                        }
+                    };
+
+                base.SetView(new StackLayout
+                    {
+                        Orientation = StackOrientation.Vertical,
+                        VerticalOptions = LayoutOptions.Start,
+                        Children =
+                            { 
+                                _sliderLabel,
+                                new StackLayout
+                                {
+                                    Orientation = StackOrientation.Horizontal,
+                                    HorizontalOptions = LayoutOptions.FillAndExpand,
+                                    Children = { _slider }
+                                },
+                                new StackLayout
+                                {
+                                    Orientation = StackOrientation.Horizontal,
+                                    HorizontalOptions = LayoutOptions.FillAndExpand,
+                                    Children = 
+                                        {
+                                            new Label
+                                            {
+                                                Text = _leftLabel,
+                                                HorizontalOptions = LayoutOptions.FillAndExpand,
+                                                FontSize = 15
+                                            },
+
+                                            new Label
+                                            {
+                                                Text = _rightLabel,
+                                                HorizontalOptions = LayoutOptions.End,
+                                                FontSize = 15
+                                            }
+                                        }
+                                }
+                            }
+                        });
+            }
+            else
+            {
+                // if the view was already initialized but is not enabled and has never been interacted with, there should be no tip text since the user can't do anything with the slider.
+                if (!Enabled && !_incrementalValueHasChanged)
+                {
+                    _slider.Value = _minimum;
+                    _sliderLabel.Text = GetLabelText(index) + "  No value selected.";
+                }            
+            }
+
+            return base.GetView(index);
+        }
+
+        private double GetIncrementalValue(double value)
+        {
+            return Math.Round(value / _increment) * _increment;
+        }
+
+        public override string ToString()
+        {
+            return base.ToString() + " -- " + _minimum + " to " + _maximum;
+        }
+    }
+}
+
```

Comment:
Got it. I would add two things to the current `NumberSliderInput`:

1) A `TipText` property, which allows the designer to specify an optional message that is displayed before the user interacts with the slider. If no such message is provided, only the label is displayed. This replaced the current `tipText` variable.

2) A `DisplaySliderValue` property, which governs whether the value of the slider is displayed after the label upon interaction with the slider. Just like the current implementation, if a tip text is specified, it is replaced with the slider value after interaction with the slider -- as long as `DisplaySliderValue` is `true`.

Does this cover your use cases?